featuretools/primitives/standard/transform/natural_language/number_of_common_words.py

```diff
@@ -46,9 +49,11 @@ class NumberOfCommonWords(TransformPrimitive):
 
     def __init__(
         self,
-        word_set=set(common_words_1000),
```

Comment:
Having a mutable default parameter is unsafe

Reply:
Leaving this alone for now because it triggers serialization test failures, but I think we should fix in future